### PR TITLE
Buttons for services control.

### PR DESCRIPTION
**Pull request by kekek2**

### CODE REVIEW

File: src/opnsense/www/js/opnsense_ui.js [hunk 1]
```diff
@@ -153,6 +153,43 @@ function updateServiceStatusUI(status) {
     $('#service_status_container').html(status_html);
 }
 
+function updateServiceControlUI(ServiceName)
```
Comment (fabianfrz) [inline src/opnsense/www/js/opnsense_ui.js]: variables should not start with a uppercase character -> would indicate a class name

File: src/opnsense/www/js/opnsense_ui.js [hunk 2]
```diff
@@ -153,6 +153,43 @@ function updateServiceStatusUI(status) {
...
+                            $("#startService").removeClass("btn-danger").removeClass("btn-success");
+                            if (data['status'] == "running") {
+                                $("#startService").addClass("btn-success");
+                            } else if (data['status'] == "stopped") {
+                                $("#startService").addClass("btn-danger");
+                            }
+                        });
+                    });
+                });
+            })((commands[i]));
```
Comment (fabianfrz) [inline src/opnsense/www/js/opnsense_ui.js]: too many brackets

File: src/opnsense/www/js/opnsense_ui.js [hunk 3]
```diff
@@ -153,6 +153,43 @@ function updateServiceStatusUI(status) {
...
+        }
+
+        var buttons = '<span id="startService" class="glyphicon glyphicon-play btn ' + status_html + '"></span>';
+        buttons += '<span id="restartService" class="glyphicon glyphicon-refresh btn"></span>';
+        buttons += '<span id="stopService" class="glyphicon glyphicon-stop btn"></span>';
+        $('#service_status_container').html(buttons);
+
+        var commands = ["start", "restart", "stop"];
+        for (var i = 0; i < commands.length; i++) {
+            (function (command) {
```
Comment (fabianfrz) [inline src/opnsense/www/js/opnsense_ui.js]: is that function really needed?

Comment (kekek2) [reply src/opnsense/www/js/opnsense_ui.js]: Variable Commands[i] used in call-back function. The value "i" will always be 3.

Comment (fabianfrz) [reply src/opnsense/www/js/opnsense_ui.js]: This does not work?
```js
commands.forEach(function(command) {
// do something
});
```

**Comment by AdSchellevis** [comment]
Wrapping service control in updateServiceControlUI(), although I’m not sure it’s a good idea to automatically depend on the processing modal dialog. Could be optional or the modal could have an other name

**Comment by fichtner** [comment]
I'm merging this, but it needs an extensive rework for a consistent look and feel with the legacy pages...

**Comment by fichtner** [comment]
Thanks!

**Pull request merged**